### DISCUSSION TITLE
Is it worth updating?

### DISCUSSION THREAD

**Discussion by Holyniwa** [vladmandic/sdnext#880] [Q&A]
Ima be real, I haven't updated automatic since the 23rd of last month, and seeing as how it's been updated to the new stuff, I got Automatic1111 massive update trauma because of all the issues popping up. program does what i need it to do without any issues so far, so I'm not sure if updating (to potentially run into issues) is even worth the anxiety...

**Comment by vladmandic** [comment]
LOL - totally your choice, I have no issues with that.
The only problem is if you do run into a problem and I manage to fix it, fix will be only in latest update.
But if it works and you don't care about any of the new features, all good.
